modules/getting-started-cli-granting-permissions.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-cli.adoc
+
+[id="getting-started-cli-granting-permissions_{context}"]
+= Granting view permissions
+
+{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the Pods. {product-title} uses and injects this service account into every Pod that launches.
+
+The following procedure creates a Service Account named `sa-user-account`. The service account communicates with the {product-title} API to learn about Pods, Services, and resources within the project.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You have a deployed image.
+* You must have `cluster-admin` privileges.
+
+// .Procedure
+
```

Comment:
Reviewers, 
Please look at the Web Console version of this content and let me know what commands would work to accomplish the same task? I wasn't able to get something working myself.

Reply:
@sagidlow, it would be something like:

```
$ oc create serviceaccount sa-user-account -n user-getting-started
$ oc adm policy add-role-to-user view -z sa-user-account -n user-getting-started
```

Reply:
@sagidlow, double-checking the Web Console part of this, I guess something is missing regarding creating the Service Account. Also, the steps defined to grant view permissions are wrong, because they're based on the Developer perspective, not the Administrator perspective. In addition, the Service Account creation are missed in both parts, Web Console and Command-Line. The steps provided are only related to grant a role binding permission to a Service Account that supposed to exist, but it doesn't. Furthermore, the Subject Name specified (`default`) already exists by default when you create a new namespace, and its tied to Docker.

So, based upon these observations, I'd suggest you to change both the Web Console and the Command-Line pages to first include the steps for the service account creation. All the necessary command-line commands I've already provided above. For the Web Console part, it would be something like:

### Creating a service account

OpenShift Container Platform automatically creates a few special service accounts in every project. The default service account takes responsibility for running the Pods. OpenShift Container Platform uses and injects this service account into every Pod that launches.

The following procedure creates a Service Account named sa-user-account. The service account communicates with the OpenShift Container Platform API to learn about Pods, Services, and resources within the project.

**_Prerequisites_**

- You are logged in to the OpenShift Container Platform web console.
- You have a deployed image.
- You are in the *Administrator* perspective.
- You must have `cluster-admin` privileges.

**_Procedure_**

1. From the **User Management** view in the Administrator perspective, click the **Service Accounts** menu.
2. Click **Create ServiceAccount**.
3. Create by manually entering YAML or JSON definitions, or by dragging and dropping a file into the editor.

    ```
    apiVersion: v1
    kind: ServiceAccount
    metadata:
         name: sa-user-account
         namespace: user-getting-started
    ```
6. Click **Create**.